Things.md

```diff
@@ -0,0 +1,111 @@
+This is a general guide about Things for all FullScreenShenanigans projects. This guide will show you how in-game Things are stored and manipulated. Everything you see in the game (trees, blocks, the player, etc.) is a Thing.  
+The Thing class is subclassed by a new class for everything (Tree class, Block class, Player class, etc.). These Things can be manipulated to change their location and properties.
+  
+### Table of Contents  
+1. [GroupHoldr](#groupholdr)
+    1. [Getting Groups](#getting-groups)
+2. [Manipulating Things](#manipulating-things)
+    1. [Shifting Things](#shifting-things)
+    2. [Opacity](#opacity)
+
+# GroupHoldr  
+
+GroupHoldr is a module that's used to store similarly typed information, such as Things, based on a shared property. Each Thing has a `groupType` string property that determines what group it's considered to be in. For example, the groups in `FSM` are:
+
+* Text
+* Character
+* Solid
+* Scenery 
+
+GroupHoldr contains an Array for each of the groups; each Array contains all the Things of that type currently in the game. Things are added to their respective group when added to the game, and removed when they die. For example, in `FSM`, solid objects like floors and blocks would be stored in the **Solid** group. 
+
+### Getting Groups  
+The groups are accessible both by static name and via passing in a String:
+
+```javascript
+// Returns the Solid group
+this.getGroup("Solid");
+
+// Returns the first Solid
+this.getGroup("Solid")[0];
+this.getSolid(0);
+```
+
+More can be read on GroupHoldr on its [Readme](https://github.com/FullScreenShenanigans/GroupHoldr/blob/master/README.md).
+
+# Manipulating Things  
+
+## Shifting Things  
+A Thing's placement on the screen is based on its `top`, `right`, `bottom`, and `left` properties. These define the bounding box for a Thing. A Thing is basically just a rectangle, with its height, width, and location defined by these properties.  
+The coordinate system in which these Things are placed has an origin at the top left corner (0,0). Things are also scaled from these raw measurements to actual in-game measurements using a predefined `unitsize` (i.e. `unitsize` of 1 means no scaling).    
+There are a number of ways to move Things, all of which are done by manipulating these 4 properties.  
+
+#### Shift  
+The first is to use the shift functions `shiftVert` and `shiftHoriz`. These move the Thing up and down or left and right.
+```javascript
+// Shifts a Thing vertically by 1.
+shiftVert(thing, 1);
+
+// Shifts a Thing horizontally by 1. 
+shiftHoriz(thing, 1);
+```
+
+#### Set  
+The next is to use a `set` function (`setTop`, `setRight`, `setBottom`, or `setLeft`). These move the Thing to line up with the bounding property.
```

Comment:
You use "The next is to" a lot. I don't like "next" in this context because it's saying these are a list of things to do, while in reality they're a set of options. Try starters like "You can" or "X does Y"